tests/test_devices.py

```diff
@@ -15,12 +15,14 @@ def format_timezone(timezone):
 @pytest.mark.django_db
 def test_device_list_basic(client, django_assert_num_queries):
     today = timezone.now()
+    uuid1 = uuid.UUID("aaaaaaa1-1342-413b-8e89-db802b2f83f6")
+    uuid2 = uuid.UUID("ccccccc3-1342-413b-8e89-db802b2f83f6")
```

Comment:
nit: you can use strings directly, should work both for factories and expected dict.